vhdbuilder/packer/components.json

```diff
@@ -42,8 +42,8 @@
       "multiArchVersions": [
         "v1.4.50",
         "v1.4.52",
-        "v1.5.11",
```

Comment:
do we not need to update vhdbuilder/packer/install-dependencies.sh as https://github.com/Azure/AgentBaker/pull/4011/files?